### DISCUSSION TITLE
infrastructure/proxmox/windows-vm-configuration/

### DISCUSSION THREAD

**Discussion by giscus** [fire1ce/3os.org#66] [General]
# infrastructure/proxmox/windows-vm-configuration/

Windows 10 or Windows 11 Virutal Machines with VirtIO Disks and Networking using Proxmox guide. Supports Proxmox's GPU passthroughs feature.

https://3os.org/infrastructure/proxmox/windows-vm-configuration/

**Comment by allenwalker3** [comment]
VirtIO SCSI is newer and better than Virtio Block . See https://pve.proxmox.com/wiki/Windows_10_guest_best_practices
